src/client/eppo-client.spec.ts

```diff
@@ -208,6 +214,158 @@ describe('EppoClient E2E test', () => {
       jest.restoreAllMocks();
     });
 
+    describe('get*AssignmentDetails', () => {
```

Comment:
For now, I'm adding unit tests here so that we don't break tests for other SDKs that do not have this functionality.

Reply:
Consider breaking it out, just for the organization's sake, into a file like `eppo-client-assignment-details.spec.ts` to keep files manageable. I'm doing the same for bandits.